packages/shared/lib/app.ts

```diff
@@ -87,6 +87,10 @@ export const logout = (): Promise<void> =>
         const ap = get(activeProfile)
```

Comment:
Should we rename this variable to _activeProfile like you have done in other places?

Reply:
Yes we should 😅